src/App.tsx

```diff
@@ -1,6 +1,52 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.css';
+import Pagination from './components/Pagination';
 
-const App: React.FC = () => <h1>Pagination</h1>;
+const App: React.FC = () => {
+  const [page, setPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
+  const [total] = useState(42);
+  const [inf, setInf] = useState(`${1}-${perPage} of ${total}`);
+
+  const onPageChange = (pageFromComponent: number) => {
+    setPage(pageFromComponent);
+  };
+
+  const onPerPageChange = (perPageFromComponent: number) => {
+    setPerPage(perPageFromComponent);
+  };
```

Comment:
Read more about handler naming - https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364
```suggestion
  const handlePageChange = (pageFromComponent: number) => {
    setPage(pageFromComponent);
  };

  const handlePerPageChange = (perPageFromComponent: number) => {
    setPerPage(perPageFromComponent);
  };
```